# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/ScreenHelperGTK.h
# Commit: dc912741adc2
# Full Hash: dc912741adc2a369f0e12d44a262c225a09cfbb5
# Author: Narcis Beleuzu <nbeleuzu@mozilla.com>
# Date: 2025-06-12 10:40:24
# Regressor Bug: 1971161
# File Overlap Count: 1
# Description:
#   Revert "Bug 1971161 [Wayland] Get HDR monitor info async r=emilio" for causing Bug 1971697 This reverts commit 49407f49202c1488708a554d528feccf4a3f85a0.
#   
#   This reverts commit adc14d3cd7e52027fc2649c7dd9f69cf055d978c.
# ==============================================================================

diff -r 536dc07bdeae -r dc912741adc2 widget/gtk/ScreenHelperGTK.h
--- a/widget/gtk/ScreenHelperGTK.h	Wed Jun 11 17:54:59 2025 -0400
+++ b/widget/gtk/ScreenHelperGTK.h	Thu Jun 12 13:36:40 2025 +0300
@@ -15,8 +15,6 @@
 
 namespace mozilla::widget {
 
-class ScreenGetterGtk;
-
 class ScreenHelperGTK final : public ScreenManager::Helper {
  public:
   ScreenHelperGTK();
@@ -25,14 +23,6 @@
   static int GetMonitorCount();
   static gint GetGTKMonitorScaleFactor(gint aMonitorNum = 0);
   static RefPtr<widget::Screen> GetScreenForWindow(nsWindow* aWindow);
-
-  static void RequestRefreshScreens();
-  static int GetLastSerial() { return gLastSerial; }
-
- private:
-  static GdkWindow* sRootWindow;
-  static StaticRefPtr<ScreenGetterGtk> gLastScreenGetter;
-  static int gLastSerial;
 };
 
 }  // namespace mozilla::widget
